charts/stable/unifi/Chart.yaml

```
@@ -21,4 +21,4 @@ sources:
 - https://github.com/jacobalberty/unifi-docker
 - https://unifi-network.ui.com
 type: application
-version: 6.11.14
+version: 6.11.15
```

This is a feature addition, hence a minor increase...
```suggestion
version: 6.12.0
```